src/coordinator/roles.js

```diff
@@ -46,6 +46,102 @@ class Roles {
     this._sdk = sdk;
   }
 
+  /**
+   * Add an application to a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: POST
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} applicationId The UUID formatted ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
```

Comment:
Is this really undefined? If it is, you should leave off the `console.log(roleApplication)` in the example, and skip the `.then`

Reply:
thx...done